.buildkite/scripts/steps/artifacts/cloud.sh

```diff
@@ -12,28 +12,32 @@ mkdir -p target
 
 download_artifact "kibana-$FULL_VERSION-linux-x86_64.tar.gz" ./target --build "${KIBANA_BUILD_ID:-$BUILDKITE_BUILD_ID}"
 
-node scripts/build \
-  --skip-initialize \
-  --skip-generic-folders \
-  --skip-platform-folders \
-  --skip-archives \
-  --docker-images \
-  --skip-docker-ubi \
-  --skip-docker-ubuntu \
-  --skip-docker-contexts
-
-docker load --input target/kibana-cloud-$FULL_VERSION-docker-image.tar.gz
-
 TAG="$FULL_VERSION-$GIT_COMMIT"
-KIBANA_BASE_IMAGE="docker.elastic.co/kibana-ci/kibana-cloud:$FULL_VERSION"
 KIBANA_TEST_IMAGE="docker.elastic.co/kibana-ci/kibana-cloud:$TAG"
 
-docker tag "$KIBANA_BASE_IMAGE" "$KIBANA_TEST_IMAGE"
-
 echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
 trap 'docker logout docker.elastic.co' EXIT
 
-docker push "$KIBANA_TEST_IMAGE"
+set +e
+DISTRIBUTION_EXISTS=$(docker manifest inspect $KIBANA_TEST_IMAGE &> /dev/null; echo $?)
+set -e
+
+if  [ $DISTRIBUTION_EXISTS -eq 0 ]; then
+  echo "Distribution already exists, skipping build"
+else
```

Comment:
@brianseeders can correct me if I'm wrong, but I think you can write this as

```suggestion
if  docker manifest inspect $KIBANA_TEST_IMAGE &> /dev/null; then
  echo "Distribution already exists, skipping build"
else
```

Reply:
Yep, looks good to me @spalger